config/initializer/redis.rb

```diff
@@ -0,0 +1 @@
+Redis.current = Redis.new(SidekiqMonitoring.redis_url)
```

Comment:
can't this be: 
`Redis.current ||= Redis.new(SidekiqMonitoring.redis_url) `
So we can use the applications `Redis.current` if set?

Reply:
`Redis.current` actually calls `Redis.new` if it is not set before and does memoizing itself https://github.com/redis/redis-rb/blob/50d0af509cf4baf01504c2f2000545c4473ea3c4/lib/redis.rb
So passing the application's redis_url and setting `Redis.current` basically is using the application's `Redis.current` :)

Reply:
Thats not what I meant. You do not want to override a `Redis.current` already set by another part of the application that includes this gem.